scripts/gdx/gdxAirdropCalculation.ts

```diff
@@ -198,3 +198,180 @@ export const airdrop = (ethers: typeof Ethers, ethSnapshotBlock) => {
 
   return { buildMerkleTree, collectAirdropData, getProof };
 };
+
+export const airdropNew = (ethers: typeof Ethers, ethSnapshotBlock) => {
+  const getBuyingAddresses = async (addresses = {}, isContracts = {}) => {
+    const provider = new ethers.providers.InfuraProvider();
+
+    let newReserve = await ethers.getContractAt(
+      [
+        "event TokenPurchased(address indexed caller,address indexed inputToken,uint256 inputAmount,uint256 actualReturn,address indexed receiverAddress)",
+        "event TokenSold(address indexed caller,address indexed outputToken,uint256 gdAmount,uint256 contributionAmount,uint256 actualReturn,address indexed receiverAddress)"
```

Comment:
TokenSold to be removed and replaced with an exchangehelper call, similar to the first airdrop that's above